doc/lua_api.md

```diff
@@ -7483,17 +7481,15 @@ child will follow movement and rotation of that bone.
         * Fourth column: subject looking to the right
         * Fifth column: subject viewed from above
         * Sixth column: subject viewed from below
-* `get_entity_name()`
-  > **Warning**: Deprecated, will be removed in a future version, use the field `self.name` instead
+* `get_entity_name()`: (**Deprecated**: Will be removed in a future version, use the field `self.name` instead)
 * `get_luaentity()`
 
 #### Player Only (No-Op for Other Objects)
```

Comment:
The syntax highlighting here is off. Could this be due to an unclosed Lua code block?

Reply:
Weird, this doesn't show in VS Code. I don't see any open code blocks, either.
![image](https://github.com/minetest/minetest/assets/90872694/d792f025-edab-4aac-8855-a6e7298981c5)

Reply:
Could also be a GitHub bug then.